usl_pipeline/cloud_functions/main_test.py

```diff
@@ -1228,6 +1241,7 @@ def test_collapse_city_cat_output_chunks(mock_firestore_client):
                     ),
                     "x_index": 0,
                     "y_index": 1,
+                    "in_validation_set": False,
```

Comment:
Your tests still use `in_validation_set` in some places.

Reply:
yep, I've now fixed this and the build is happy